packages/material-ui/src/Tabs/TabIndicator.test.js

```diff
@@ -14,28 +15,28 @@ describe('<TabIndicator />', () => {
   const style = { left: 1, width: 2 };
 
   before(() => {
-    shallow = createShallow({ dive: true });
     classes = getClasses(<TabIndicator {...defaultProps} />);
   });
 
   it('should render with the root class', () => {
-    const wrapper = shallow(<TabIndicator {...defaultProps} />);
-    expect(wrapper.name()).to.equal('span');
-    expect(wrapper.hasClass(classes.root)).to.equal(true);
+    const { container } = render(<TabIndicator {...defaultProps} />);
+    expect(container.firstChild).to.have.tagName('span');
+    expect(container.firstChild).to.have.class(classes.root);
   });
 
   describe('prop: style', () => {
     it('should be applied on the root element', () => {
-      const wrapper = shallow(<TabIndicator {...defaultProps} style={style} />);
-      expect(wrapper.props().style).to.equal(style);
+      const wrapper = mount(<TabIndicator {...defaultProps} style={style} />);
+      const modal = wrapper.find(TabIndicator);
+      expect(modal.props().style).to.equal(style);
```

Comment:
We should be able to test this using render. Passing in style (inline) and checking it exists on the dom element.

Reply:
Ok, which of the approaches is better in your opinion?

```
const tab = container.firstChild;
const { width, left } = tab.style
expect(left).to.equal('1px');
expect(width).to.equal('2px');
```

or
`expect(container.querySelectorAll('span')[0].style).not.to.equal(null)`

Reply:
maybe?

```js
expect(tab.style).to.have.property('left', '1px');
expect(tab.style).to.have.property('width', '2px');
```